Tests/TestHelpers.swift

```diff
@@ -1,12 +1,12 @@
 import Moya
-import Foundation
 
-extension String {
-    var urlEscaped: String {
-        return self.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)!
-    }
-}
+#if os(OSX)
+import AppKit
+#else
+import Foundation
```

Comment:
https://developer.apple.com/documentation/swift/int/1455396-init
->
**Framework**
Core Graphics

Documentation is* awesome. 😬


\* And with is, I mean *can be*.